elasticdl/python/tests/embedding_test_module.py

```diff
@@ -4,23 +4,83 @@
 from elasticdl.python.elasticdl.layers.embedding import Embedding
 
 
-class CustomModel(tf.keras.Model):
-    def __init__(self, output_dim=16):
-        super(CustomModel, self).__init__(name="embedding_test_model")
+class EdlEmbeddingModel(tf.keras.Model):
+    def __init__(self, output_dim=16, weights=None):
+        """
+        Arguments:
+            output_dim: An Integer. It is the output dimension of embedding
+                layers in `EdlEmbeddingModel`.
+            weights: A numpy ndarray list. Unless `weights` is None, dense
+                layer initializes its weights using `weights`.
+        """
+        super(EdlEmbeddingModel, self).__init__(
+            name="test_model_with_edl_embedding"
+        )
         self.output_dim = output_dim
+        if weights:
+            if len(weights) != 2:
+                raise ValueError(
+                    "test_model_with_edl_embedding constructor receives "
+                    "weights with length %d, expected %d" % (len(weights), 2)
+                )
+
         self.embedding_1 = Embedding(output_dim)
         self.embedding_2 = Embedding(output_dim)
         self.concat = Concatenate()
-        self.dense = Dense(1, input_shape=(output_dim * 3,))
+        self.dense = Dense(1, weights=weights)
+        self.flatten = Flatten()
+
+    def call(self, inputs, training=False):
+        f1 = self.embedding_1(inputs["f1"])
+        f2 = self.embedding_1(inputs["f2"])
+        f3 = self.embedding_2(inputs["f3"])
+        x = self.concat([f1, f2, f3])
```

Comment:
We don't need to assign f1, f2, f3. You can use use them directly in concat.

Reply:
Fixed.